composer.json

```diff
@@ -55,6 +57,7 @@
     "drupal/single_user_role": "^1.0@beta",
     "drupal/smsframework": "^2.2@RC",
     "drupal/taxonomy_unique": "^2.5",
+    "drupal/textfield_counter": "^2.3",
```

Comment:
Are we using this module? I can't see this module's entry in any info.yml file

Reply:
Removed!!